src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.vcxproj

```diff
@@ -69,13 +71,16 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\dep\jsoncpp\json;$(OpenConsoleDir)src\inc;$(OpenConsoleDir)src\inc\test;$(WinRT_IncludePath)\..\cppwinrt\winrt;"$(OpenConsoleDir)\src\cascadia\TerminalApp\lib\Generated Files";%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <PrecompiledHeaderFile>precomp.h</PrecompiledHeaderFile>
+      <!-- <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\dep\jsoncpp\json;$(OpenConsoleDir)src\inc;$(OpenConsoleDir)src\inc\test;$(WinRT_IncludePath)\..\cppwinrt\winrt;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories> -->

```

Comment:
Do we want to remove these commented `AdditionalIncludeDirectories` and `AdditionalOptions`?